reset_test_db.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Can we add some information about what the script is for here? Maybe we can update the CONTRIBUTING.md to mention about these scripts?

Reply:
Added explanation info to the `CONTRIBUTING.md`